internal/server/create_draft_test.go

```diff
@@ -246,14 +243,11 @@ func TestPostCreateDraftWhenValidationError(t *testing.T) {
 		Return(sirius.User{Roles: []string{"private-mlpa"}}, nil)
 	client.
 		On("RefDataByCategory", mock.Anything, sirius.CountryCategory).
-		Return([]sirius.RefDataItem{{Handle: "GB", Label: "Great Britain"}}, nil)
+		Return([]sirius.RefDataItem{}, nil)
```

Comment:
Worth keeping this to make sure the list of the countries from the API (which this is mocking) is passed to the template